### PR TITLE
calling_at change

### PR DESCRIPTION
**Pull request by michaelmcarthur**
Using 'calling_at' rather than 'destination' allows the user to chose any station on the route rather than the end of route destination.

### CODE REVIEW

**Comment by robmarkcole** [comment]
To be consistent could you change self._destination_name to self._calling_at and change all references to destination_name to calling_at?

File: HASS files/uk_transport.py [hunk 1]
```diff
@@ -47,7 +47,7 @@
         vol.Required(CONF_BUS_DIRECTION): cv.string}],
     vol.Optional(CONF_LIVE_TRAIN_TIME): [{
         vol.Required(ATTR_STATION_CODE): cv.string,
-        vol.Required(ATTR_DESTINATION_NAME): cv.string}]
+        vol.Required(ATTR_calling_at): cv.string}]
```
Comment (robmarkcole) [inline HASS files/uk_transport.py]: attributes should be capitalised, so ATTR_CALLING_AT

**Comment by michaelmcarthur** [comment]
added in the changes you requested. the 'create uk_transport.py' is a fix for me removing 'query returns an error' from the master code.

**Comment by michaelmcarthur** [comment]
in the configuration.yaml the destination needs to be changed to  calling_at

**Pull request merged**